src/components/i18nStrings.vue

```diff
@@ -103,6 +103,8 @@ export default {
 			if (!lang || lang in this.state) return
 			this.$set(this.state, lang, '')
 			// wait for rendering so that the ref is present in dom before focus
+			
+			this.$store.commit('setLanguages', {[lang]:true})
```

Comment:
For this line, I'd suggest switching places with the "wait for rendering" comment line because the comment is referring to the this.$nextTick line.